apps/bare-expo/package.json

```diff
@@ -26,7 +26,10 @@
         "expo-ads-admob",
         "expo-module-template",
         "expo-module-template-local",
-        "expo-face-detector"
+        "expo-face-detector",
+        "expo-dev-client",
+        "expo-dev-launcher",
+        "expo-dev-menu"
```

Comment:
from pr description it says the expo-dev-client support pr was landed. do we still need to disable dev-client?

Reply:
dev-client will be reenabled in a follow-up PR

Reply:
I ended up fixing everything in this same PR to facilitate CI getting green